src/flb_engine.c

```diff
@@ -754,6 +754,11 @@ int flb_engine_start(struct flb_config *config)
     flb_engine_evl_set(evl);
 
     /* Start the Logging service */
+    config->log_metrics_ctx = flb_log_metrics_create();
```

Comment:
Not sure about whether to clean these up or where. `config->storage_metrics_ctx` is similar and it is never cleaned up.